cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php

```diff
@@ -327,6 +347,29 @@ public function filter_video_tags( $content ) {
 					$new_tag = str_replace( 'src="' . $url . '"', 'id="cloudinary-video-' . esc_attr( $instance ) . '"', $tag );
 					$content = str_replace( $tag, $new_tag, $content );
 				} else {
+					$res         = wp_remote_head( $cloudinary_url );
```

Comment:
@dugajean I have some concerns with doing this call every time on a video, regardless of having transformations or not. Even though it's just a HEAD, it will add significant traffic on a busy site.
I'll look for some ideas and add to the ticket